integration_test/third_party_apps_test.go

```diff
@@ -73,6 +73,19 @@ func osFolder(platform string) string {
 	return "linux"
 }
 
+// rejectDuplicates looks for duplicate entries in the input slice and returns
+// an error if any is found.
+func rejectDuplicates(apps []string) error {
+	unique := make(map[string]bool)
+	for _, app := range apps {
+		if unique[app] {
+			return fmt.Errorf("application %q appears multiple times in supported_applications.txt", app)
```

Comment:
[Optional] Would it make sense to list all duplicates?

Reply:
I guess by resolving this you were saying that reporting one duplicate at a time is fine. Well, it was optional anyway. For the record, I was suggesting something like:
```
func rejectDuplicates(apps []string) error {
	seen := make(map[string]bool)
	var dups []string
	for _, app := range apps {
		if seen[app] {
			dups = append(dups, app)
		}
		seen[app] = true
	}
	if len(dups) > 0 {
		return fmt.Errorf("applications %q appear multiple times in supported_applications.txt", dups)
	}
	return nil
}
```